resources/assets/js/app/services/SubscriptionService.js

```diff
@@ -0,0 +1,19 @@
+import {securedRequest} from 'app/services/RequestService';
+
+export const getSubscriptions = () => {
+    return securedRequest.get('/api/v1/subscriptions');
+};
+
+export const changeSubscriptionStatus = (id, status) => {
+    return securedRequest.put(`/api/v1/subscriptions/${id}/status`, {
+        status: !!status
```

Comment:
property name should be equals to property that you going to update in your case as I understood it will be is_active, not status